core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala

```diff
@@ -122,14 +124,14 @@ private[spark] class HeartbeatReceiver(sc: SparkContext, clock: Clock)
       context.reply(true)
 
     // Messages received from executors
-    case heartbeat @ Heartbeat(executorId, accumUpdates, blockManagerId, executorMetrics) =>
+    case heartbeat @ Heartbeat(executorId, accumUpdates, blockManagerId, executorUpdates) =>
```

Comment:
What was the reason for this variable name changes?

Reply:
The name of the field in the Heartbeat case class was changed, so it made sense to change the variable used in this pattern matching to match the field name.
The field used to be of type ExecutorMetrics, so the name was executorMetrics. It is now a Map[(Int, Int), ExecutorMetrics], so I call it executorUpdates instead.